DatabaseOperations/DataTransferObjects/ConnectionOptions.cs

```diff
@@ -98,6 +99,14 @@ internal bool IsValid()
             return _isValid;
         }
 
+        internal void RemovePathFromBackupLocation()
+        {
+            var startOfPath = BackupLocation.SubstringBeforeValue(DatabaseName);
+            var requiredLocation = BackupLocation.SubstringAfterValue(startOfPath);
+            BackupLocation = requiredLocation;
+            _executionParameters = GetParameters(DatabaseName, BackupLocation, Description);
```

Comment:
Not sure about this, as it looks like a code smell.

OK for now, but will need looking at later.  I'll create a new issue for it.